src/DataModel/IApiDocumentation.ts

```diff
@@ -0,0 +1,9 @@
+import {IMetadataDescription} from "./IMetadataDescription";
+import {IData} from "./IData";
+
+export interface IApiDocumentation extends IMetadataDescription
+{
+    entryPoint: string;
+
+    getEntryPoint(): Promise<IData>;
```

Comment:
I don't like how this is the functionality of the documentation. I treat the `ApiDocumentation` itself as a resource. Besides, this smells like it would violate Single Responsibility Principle

What would you think about smth like

``` js
IHydraApi api = new HydraApi("http://entrypoint");

IApiDocumentation docs = api.getApidDocumentation();
IResource entry = api.getEntryPoint();
```